src/arch/riscv/config.cmake

```diff
@@ -48,6 +50,12 @@ if (KernelSel4ArchRiscV32)
     set(KernelPTLevels 2 CACHE STRING "" FORCE)
 endif()
 
+config_option(KernelBuildRocketChipZedboard BUILD_ROCKET_CHIP_ZEDBOARD "Build application to \
```

Comment:
If it is specific to the Spike platform, this option can be moved to src/plat/spike/config.cmake.
Additionally, renaming it to something like KernelPlatformSpikeRocketChip will be more consistent with how other platforms and boards are named.

Reply:
Yeah, it is specific to the Spike platform. I'll move it to the suggested location and update the cmake option, leaving the `BUILD_ROCKET_CHIP_ZEDBOARD` output alone.